core/src/edu/cornell/gdiac/main/PassThroughPlatformModel.java

```diff
@@ -86,23 +92,28 @@ protected void createFixtures() {
       return;
     }
 
-    // Create the sensor fixtures
-    FixtureDef bodyFixtureDef = bodySensor != null ? bodySensor.getFixtureDef() : null;
-    if (bodyFixtureDef != null) {
-      bodyFixture = body.createFixture(bodyFixtureDef);
-      bodyFixture.setUserData(bodySensor);
-    }
-
-    FixtureDef bottomFixtureDef = bottomSensor != null ? bottomSensor.getFixtureDef() : null;
-    if (bottomFixtureDef != null) {
-      bottomFixture = body.createFixture(bottomFixtureDef);
-      bottomFixture.setUserData(bottomSensor);
-    }
-
     assert bodySensor != null;
-    assert bodyFixtureDef != null;
     assert bottomSensor != null;
+    assert sideSensor != null;
+
+    // Create the sensor fixtures
+    FixtureDef bodyFixtureDef = bodySensor.getFixtureDef();
+    assert bodyFixtureDef != null;
+    bodyFixture = body.createFixture(bodyFixtureDef);
+    bodyFixture.setUserData(bodySensor);
+
+    FixtureDef bottomFixtureDef = bottomSensor.getFixtureDef();
     assert bottomFixtureDef != null;
+    bottomFixture = body.createFixture(bottomFixtureDef);
+    bottomFixture.setUserData(bottomSensor);
+
+    FixtureDef leftFixtureDef = sideSensor.getFixtureDef();
+    assert leftFixtureDef != null;
+    sideFixture = body.createFixture(leftFixtureDef);
+    sideFixture.setUserData(sideSensor);
+
+    // Set as bullet
+    body.setBullet(true);
```

Comment:
why set as bullet? Platforms don't move so they should not be a bullet, this will hurt our performance.